src/navigation/components/DesktopNavigation/components/LeftDesktopNavigation/LeftDesktopNavigation.tsx

```diff
@@ -0,0 +1,71 @@
+import React, {useCallback, useContext, useEffect} from 'react';
+
+import _ from 'lodash';
+
+import OverflowScroller from '../../../../../components/OverflowScroller/OverflowScroller';
+import {LocationContext} from '../../../../../context/locationContext';
+import {NavigationItemModel} from '../../../../../models/navigation';
+import {block} from '../../../../../utils';
+import {ItemColumnName} from '../../../../constants';
+import {getItemClickHandler} from '../../../../utils';
+import NavigationItem from '../../../NavigationItem/NavigationItem';
+
+import './LeftDesktopNavigation.scss';
+
+const b = block('left-desktop-navigation');
+
+export interface NavigationProps {
+    items: NavigationItemModel[];
+    activeItemId?: string;
+    className?: string;
+    highlightActiveItem?: boolean;
+    onActiveItemChange: (id?: string) => void;
+}
+
+const LeftDesktopNavigation: React.FC<NavigationProps> = ({
+    className,
+    onActiveItemChange,
+    items,
+    activeItemId,
+}) => {
+    const {asPath, pathname} = useContext(LocationContext);
+
+    const hidePopup = useCallback(() => {
+        onActiveItemChange();
```

Comment:
seems like we can use `onActiveItemChange` instead of `hidePopup`

Reply:
sure